backend/btrixcloud/crawlconfigs.py

```diff
@@ -597,13 +598,16 @@ async def get_crawl_configs(
         page: int = 1,
         created_by: Optional[UUID] = None,
         modified_by: Optional[UUID] = None,
+        profile_ids: Optional[List[UUID]] = None,
+        # deprecated - use profile_ids instead
```

Comment:
Think we don't actually need to deprecate this, as it was just added last week in webrecorder/browsertrix#2703